www/%team/index.html.spt

```diff
@@ -18,7 +18,9 @@ is_team_owner = not user.ANON and team.owner == user.participant.username
 {% block head %}
     <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
           href="https://gratipay.com/{{ team.slug }}/" />
+    <meta name="ROBOTS" content="NOINDEX, NOFOLLOW" />
```

Comment:
Sorry, I don't agree here. imo rejected a team just means it wasn't compatible with gratipay and what we do here. Nowhere do we reflect on how good the team is or the nature of work they do. How people perceive that rejection is really upto them. Letting people find rejected teams will at least give them the idea about what we do here and what we don't, if nothing else.